source/develop/release-management/features/network/allowExplicitDnsConfiguration.html.md

```diff
@@ -40,11 +40,30 @@ values from resolv.conf will be used. If either of them is used, it
 will be passed to VDSM overriding configuration from resolv.conf. For
 example we can set DNS configuration in a Network related to certain DC
 (or later to certain Cluster) and this configuration will override
-configuration in resolv.conf. If we now setup configuration in
+configuration in resolv.conf. Note: when Network is being attached in 
+certain host, NetworkAttachment record is created for this, and 
+dns configuration is copied from Network record to NetworkAttachment 
+record. There we can further modify it. If we now setup configuration in
```

Comment:
alright then:

- no copying. 
- updating Network, which was already attached to some host, will make all it's relevant network attachments (which does not override it) out-of-sync. User can then click sync all network to reapply dns configuration.